code/game/objects/items/tools/synth_bracer/bracer_subtypes.dm

```diff
@@ -0,0 +1,53 @@
+/obj/item/clothing/gloves/synth/wy
+	name = "PK-130C SIMI wrist-mounted computer"
+	desc = "Developed by a joint effort between Weyland-Yutani CIART and the USCM R&D Division, the SIMI portable computer is the ultimate solution for situational awareness, personnel monitoring and communication. This one has a corporate-white finish."
+
+	icon_state = "bracer_white"
+	base_item_slot_state = "bracer_white"
+	item_state_slots = list(
+		WEAR_HANDS = "bracer_white"
+	)
+
+	bracer_color = SIMI_COLOR_WHITE
+	faction = FACTION_WY
+
+/obj/item/clothing/gloves/synth/wy/pmc
+	faction = FACTION_PMC
+
+/obj/item/clothing/gloves/synth/wy/pmc/preset
+	faction = FACTION_PMC
+	actions_list_added = list(
+		/datum/action/human_action/synth_bracer/repair_form,
+		/datum/action/human_action/synth_bracer/motion_detector,
+	)
+
+/obj/item/clothing/gloves/synth/wy/pmc/preset/New()
+	underglove = new /obj/item/clothing/gloves/marine/veteran/pmc
+	underglove.forceMove(src)
```

Comment:
you don't need to forceMove() here you can just put it as argument when creating it